src/Dev/Deprecation.php

```diff
@@ -179,12 +203,12 @@ public static function get_enabled()
         // noop
     }
 
-    private static function get_is_enabled(): bool
+    public static function isEnabled(): bool
```

Comment:
Needs to be public so that it can be called by unit tests that wish to skip because they are testing deprecated code i.e.
```php
if (Deprecation::isEnabled()) {
    $this->markTestSkipped('Test calls deprecated code');
}
```